_drafts/2019-02-27-face-anti-spoofing.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+author: "Fatih Çağatay Akyön"
+title:  "Canlılık (Liveness) - Yüz Sahteciliğini Önleme (Face Anti-Spoofing) Hakkında Bilmeniz Gerekenler"
+description: "Bu çalışma, Fatih Çağatay Akyön tarafından kaleme alınan https://medium.com/codable/face-anti-spoofing-starter-kit-f248ed195a3c Medium makalesinin Türkçe çevirisidr."
+date:   2019-02-27
+categories: yuz-tanima
+tags: ["yuz-tanima"]
+permalink: /blog/:categories/:title
+
+prev-page-url: /blog/yuz-tanima/yuz-sahteciligini-onleme
```

Comment:
bunu önceki/sonraki yazılara buttonlar ile yönlendirme yapmak için ekliyoruz. bu tek yazı olduğu için kaldırabiliriz.
devamı yazısı olduğunda daha sonra ekleyebiliriz